test/utils/Murky.sol

```diff
@@ -27,8 +27,8 @@ abstract contract MurkyBase {
         // proof length must be less than max array size
         bytes32 rollingHash = valueToProve;
         uint256 length = proof.length;
-        unchecked {
-            for (uint256 i = 0; i < length; ++i) {
+        for (uint256 i = 0; i < length; ++i) {
+            unchecked {
```

Comment:
this must be left the same as the forked contract: https://github.com/dmfxyz/murky/blob/main/src/common/MurkyBase.sol

we've hardcoded this contract in our codebase because we didn't want to add more dependencies

Reply:
I see. Got it. Thanks for the review.

Reply:
Agree with @andreivladbrg